js/latoken.js

```diff
@@ -197,7 +197,7 @@ module.exports = class latoken extends Exchange {
         return result;
     }
 
-    async fetchCurrencies (params = {}) {
+    async fetchCurrencies (symbol = undefined, params = {}) {
```

Comment:
We can't really change the signature of this method, because it has to be unified with the other exchanges. Neither of them really have a `symbol` argument there. Need to think of a proper workaround for this or remove this entirely.

Reply:
In this situation, we need to remove "ExchangeInfo/pairs/{currency}", "Account/balances/{currency}", "ExchangeInfo/pairs/{currency}", "ExchangeInfo/pair"  from list of api methods in the top of file

Reply:
@PolozovT no, we always have a full list of endpoint, even if some of those are not called directly – the user can call them implicitly as described in the Manual:

- https://github.com/ccxt/ccxt/wiki/Manual#implicit-api-methods

So, in this case we need to keep the endpoints to match the exchange docs, and we need to fix the fetchCurrencies and fetchBalance for a redundant argument.